profiling/flamegraph.sh

```diff
@@ -5,15 +5,15 @@ INPUTFILE=$2
 OUTPUTFILE=$3
 
 # Record samples with perf
-perf record -g $EXECUTABLE $INPUTFILE $OUTPUTFILE
-
+# perf record -g $EXECUTABLE
+perf record --call-graph dwarf $EXECUTABLE $INPUTFILE $OUTPUTFILE
 # Convert the data to be readable for flamegraphs
 perf script > out.perf
-
+BASE_URL=https://raw.githubusercontent.com/eguiraud/FlameGraph/160b531f4c5ef0fec37e2b719ec609842a02aa99/
 # Perform the stack collapse
-curl -Os https://raw.githubusercontent.com/brendangregg/FlameGraph/master/stackcollapse-perf.pl > stackcollapse-perf.pl
+curl -Os ${BASE_URL}/stackcollapse-perf.pl > stackcollapse-perf.pl
```

Comment:
```suggestion
curl -OLs ${BASE_URL}/stackcollapse-perf.pl > stackcollapse-perf.pl
```

Needed so that it follows redirects (which it has to do).